utils/analyzers/time_tracker.rb

```diff
@@ -47,8 +48,8 @@ def initialize
         end
 
         def add(start_time, end_time)
-          start_day = start_time.to_date
-          end_day = end_time.to_date
+          start_day = start_time.to_datetime.utc.to_date
```

Comment:
I'll check it out again, but this was required to get the specs to pass after 🕗 PM. So the setup may have been incorrect? I will revisit. Whatever it is, the data needs to be faked to whatever Trello is _actually_ giving us. It was a week ago, so I need to look at it again.

I'm pretty sure the issue was it comes in as a string with the UTC information in it, but `to_date` was not taking it into account. By doing `to_datetime` to preserve the time offset, then `.utc.to_date` you got the correct date.